auto_nag/scripts/component.py

```diff
@@ -73,22 +84,31 @@ def get_bugs(self, date='today', bug_ids=[]):
         # Get the encoded component.
         indexes = probs.argmax(axis=-1)
         # Apply inverse transformation to get the component name from the encoded value.
-        components = self.model.clf._le.inverse_transform(indexes)
+        suggestions = self.model.clf._le.inverse_transform(indexes)
 
         results = {}
-        for bug, prob, index, component in zip(bugs, probs, indexes, components):
+        for bug, prob, index, suggestion in zip(bugs, probs, indexes, suggestions):
             # Skip product-only suggestions that are not useful.
-            if '::' not in component and bug['product'] == component:
+            if '::' not in suggestion and bug['product'] == suggestion:
                 continue
 
-            component = self.model.CONFLATED_COMPONENTS_MAPPING.get(component, component)
+            suggestion = self.model.CONFLATED_COMPONENTS_MAPPING.get(suggestion, suggestion)
+
+            assert '::' in suggestion
```

Comment:
Suggestion: you could use logger.error(...) and have an error report and then avoid to break everything if one suggestion doesn't contain '::'

Reply:
If '::' is not in the suggestion, something must be broken in bugbug.
If this breaks all the rest of the checkers though, I'll just report an error.